### PR TITLE
Suggested README improvements

### PR DESCRIPTION
**Pull request by vossisboss**
I feel like since a lot of beginners are likely going to come across this template that we should maintain some consistency between how the virtual environment is set up in the tutorial and this template.

I also feel that people would benefit from having the fully written out command for installing the template.

I added in a few additional links and resources as well for folks who might be using a different operating system.

Please let me know what you think. Happy to get feedback.

### CODE REVIEW

**Comment by vossisboss** [comment]
The `load-data` command is not optional because the cachetable and the migrations aren't created without using that command. If we want to make the dummy data completely optional, then we need to include a separate command for creating those things without the data.

File: README.md [hunk 1]
```diff
@@ -4,57 +4,62 @@ This Django project template is designed for creating Wagtail builds quickly, in
...
-4. **Navigate to Project Directory**: Move into the newly created project directory.
+5. **Install Wagtail**: Install the Wagtail CMS package using pip.
 
     ```bash
-    cd myproject
+    pip install wagtail
     ```
 
-5. **Install Project Dependencies**: Install the project's dependencies into a virtual environment.
+6. **Initialize Project**: Use the Django `startproject` command to create a new project based on the Wagtail Starter Kit template.
```
Comment (RealOrangeOne) [inline README.md:36]:
```suggestion
6. **Initialize Project**: Use the `wagtail start` command to create a new project based on the Wagtail Starter Kit template.
```

File: README.md [hunk 2]
```diff
@@ -4,57 +4,62 @@ This Django project template is designed for creating Wagtail builds quickly, in
...
+7. **Install Project Dependencies**: Install the project's dependencies into a virtual environment.
 
+    ```bash
     pip install -r requirements.txt
     ```
 
 All commands from now on should be run from inside the virtual environment.
 
-6. **Load Dummy Data**: Optionally load in some dummy data, to populate the site with some content.
+7. **Load Dummy Data**: Load in some dummy data to populate the site with some content.
```
Comment (RealOrangeOne) [inline README.md:50]: Nit: "7" is duplicated here - should be 8.

File: README.md (hunk 1 above)
Comment (vossisboss) [reply README.md:36]: Looks good to me!

File: README.md (hunk 2 above)
Comment (vossisboss) [reply README.md:50]: Great catch. Thank you!

**Pull request merged**